pkg/apis/servicecatalog/validation/broker.go

```diff
@@ -188,20 +188,6 @@ func validateCommonServiceBrokerSpec(spec *sc.CommonServiceBrokerSpec, fldPath *
 		)
 	}
 
-	if spec.RelistBehavior == sc.ServiceBrokerRelistBehaviorDuration && spec.RelistDuration == nil {
-		commonErrs = append(
-			commonErrs,
-			field.Required(fldPath.Child("relistDuration"), "relistDuration must be set if relistBehavior is set to Duration"),
-		)
-	}
-
-	if spec.RelistBehavior == sc.ServiceBrokerRelistBehaviorManual && spec.RelistDuration != nil {
```

Comment:
What's the defined behavior for a manual relist? I don't quite understand why your change makes it okay to say manual and set a duration.

Reply:
Manual is "only respond to an increase in the relist-requests, no automated pinging of the broker" 

My opinion is that this causes unecessary friction if switching from duration to manual, and that while it doesn't make much sense to explicitly specify manual and duration, it shouldn't be a hard error. Manual should override everything else.

Reply:
Ah, that makes sense thanks! 👍